twitter_analytics_client.rb

```diff
@@ -34,11 +34,11 @@ def get_analytics_data
   private
 
   def start_date
-    @start_date ||= (Time.now.beginning_of_month).to_i.to_s + "000"
+    @start_date ||= ( Time.now.beginning_of_month.since(9.hours)).to_i.to_s + "000"
```

Comment:
You can use `utc` method.
- TimeWithZone: https://api.rubyonrails.org/v5.2.1/classes/ActiveSupport/TimeWithZone.html#method-i-utc
- Time: https://docs.ruby-lang.org/ja/latest/class/Time.html#S_GM
```ruby
[1] pry(main)> Time.now.utc
=> 2018-09-03 09:01:06 UTC
[2] pry(main)> Time.now
=> 2018-09-03 18:01:17 +0900
```

Reply:
Thank you!
I use this.